src/components/Layout/FormPageLayout.tsx

```diff
@@ -56,27 +58,25 @@ function FormPageLayoutComponent<F>(props: FormPageLayoutProps<F>) {
     [formSections],
   );
 
-  // The grid columns are defined as: "left-gutter, left-nav, form-content, right-sidebar, right-gutter"
-  const gridColumns =
-    "minMax(0, auto) minMax(100px, 250px) minMax(350px, 1000px) minMax(min-content, 300px) minMax(0, auto)";
+  // The grid columns are defined as: "left-nav, form-content, right-sidebar"
+  const gridColumns = "minMax(100px, 250px) minMax(350px, 1000px) minMax(min-content, 300px)";
```

Comment:
Reverting back to a more conventional "max content width" container rather than having "gutter" columns based on feedback: https://github.com/homebound-team/beam/pull/1123/files/95b6e23256c1388806be3d9a40f35d9e6bc2b387#r2024882802

I originally did it this way when we weren't relying on the page scroll and needed the scroll bar to not be floating on large screens, that's not an issue any more.